.github/workflows/feg-workflow.yml

```diff
@@ -99,6 +102,18 @@ jobs:
         with:
           name: Unit Test Results
           path: "/tmp/test-results"
+      - name: Run radius src unit tests
+        if: always()
+        id: radius-src-unit-tests
+        run: |
+          cd $MAGMA_ROOT/feg/radius/src
```

Comment:
Nit: Could use `working-directory` for this.

Reply:
Done. Nice suggestion, thanks :slightly_smiling_face: